app/sidekiq/lighthouse/benefits_discovery/log_eligible_benefits_job.rb

```diff
@@ -24,8 +24,8 @@ def perform(user_uuid, service_history)
         execution_time = Time.current - start_time
         StatsD.measure(self.class.name, execution_time)
         sorted_benefits = sort_benefits(eligible_benefits)
-        formatted_benefits = sorted_benefits.to_h.to_s.tr('\"', '').tr(' ', '')
-        StatsD.increment('benefits_discovery.logging', tags: [formatted_benefits])
+        formatted_benefits = format_benefits(sorted_benefits)
+        StatsD.increment('benefits_discovery_logging', tags: ["eligible_benefits:#{formatted_benefits}"])
```

Comment:
`benefits_discovery.logging` gets translated to `benefits_discovery_logging` in DD. And eligible_benefits is the actual tag name.